src/mlpack/methods/pca/pca_main.cpp

```diff
@@ -164,6 +164,6 @@ static void mlpackMain()
   }
 
   // Now save the results.
-  if (CLI::HasParam("output"))
-    CLI::GetParam<arma::mat>("output") = std::move(dataset);
+  if (CMD::HasParam("output"))
+    CMD::GetParam<arma::mat>("output") = std::move(dataset);
```

Comment:
I tried to build and run `mlpack_pca` like we discussed this morning, using revision ed334ad17.  It built successfully (once I removed `libgfortran.a` from the list of extra libraries to link against on my x86_64 host), and when I ran it I didn't get an immediate segfault but this exception instead:

```
$ bin/mlpack_pca
terminate called after throwing an instance of 'CLI::BadNameString'
  what():  Only one positional name allowed, remove: c
Aborted
```

Is this the error you were referencing this morning or is this something else?  I'm happy to try and dig in if you like. :+1: